packages/cli/lib/types.ts

```diff
@@ -3,12 +3,11 @@ export interface GlobalOptions {
     debug: boolean;
 }
 
-export type ENV = 'local' | 'staging' | 'production';
+export type ENV = 'local' | 'cloud';
```

Comment:
does this change affect Enterprise or self-hosted installs?

Reply:
nope, it's useless, we need to remove that entirely and just enforce NANGO_HOSTPORT